app/views/users/show.html.erb

```diff
@@ -29,8 +29,22 @@
       </div>
       
       <% if @presenter.display_pre_launch? %>
-          <%= render partial: 'users/show/timeline', locals: { user: @presenter.user, scoring_pull_requests: @presenter.scoring_pull_requests, non_scoring_pull_requests: @presenter.non_scoring_pull_requests, show_timer: @presenter.show_timer?, show_congratulations: @presenter.show_congratulations? }  %>
-      <% else %>
+        <div class="container pre-launch-container">
+          <h2 class='title is-2'>Get ready to start contributing to Hacktoberfest.</h2>
+          <h4 class="title is-4">Pull requests for Hacktoberfest count October 1-31.</h4>
+          <br/>
+
+          <%= link_to "Find out more", details_path, anchor: "getting-started", class: "button-static" %>
+          <%= link_to "Attend an event", events_path, class: "button-static" %>
+          <%= link_to "Organize an event", event_kit_path, class: "button-static" %>
+        </div>
+      <% elsif scoring_pull_requests.empty? %>
```

Comment:
If the contest is active this seems to break the view?

Reply:
```suggestion
      <% elsif @presenter.scoring_pull_requests.empty? %>
```
Ah yes, missed presenter